src/pages/example.tsx

```diff
@@ -11,6 +21,79 @@ export default function Example() {
           Back to homepage
         </Button>
       </Link>
+
+      <Dropdown
+        options={[
+          {
+            value: 1,
+            label: 'Option one',
+          },
+          {
+            value: 2,
+            label: 'Option two',
+          },
+          {
+            value: 3,
+            label: 'Option three',
+          },
+        ]}
+      >
+        <Button icon={FaChevronDown} variant="secondary" iconPosition="right">
+          Menu Dropdown
+        </Button>
+      </Dropdown>
+
+      <Menu mode="horizontal">
+        <SubMenu
+          popupOffset={[-20, 0]}
+          title={<Button icon={FaTools} variant="reduced" />}
+          key="Settings1"
+          popupClassName="page-header__popup"
```

Comment:
Don't use a global class here. If you need specific styling for the example page add css modules file here.
Global should only be used for overriding the styles from external libraries. 
Also the styling should also work outside of the page-header, so increase the scope here.